.eslintrc.yaml

```diff
@@ -21,3 +21,6 @@ ignorePatterns:
   - test/fixtures/circular-a.js
   - test/fixtures/circular-b.js
   - test/fixtures/reexport.js
+  - test/fixtures/duplicate-explicit.mjs
+  - test/fixtures/duplicate.mjs
+  - test/fixtures/export-types/default-call-expression-renamed.mjs
```

Comment:
I don't see these files anywhere in the PR?

Reply:
Sorry, should have added an explanation to the PR. 

Because we use a caret version for eslint and no lock file, these files started failing with `Multiple exports of name 'foo'` [in CI](https://github.com/nodejs/import-in-the-middle/actions/runs/11050280282/job/30697711069#step:5:11) but passed locally until I nuked `node_modules` and installed again. 

So these failures were caused by a recent update in eslint!